### ISSUE TITLE
Cant Load Mod Biomes, Ores ect. and gets stuck at World Creation

### ISSUE THREAD

**Issue by I3IueJay** [JavidPack/TerraCustom#13]
Currently i am Playing Terraria with a few Mods such as Thorium, and Thorium comes with its own Ores as well as a very new Biome. I have used terracustom bevore (without mods) to create my fav worlds and everything worked just fine. Now ive read that i could also use terracustom to create worlds even with mods active and that it should work fine (ive also read that there should be extra settings for mod stuff to choose from) but it doesent seem to work tho. Ive updated the modloader, terra custom as well as all the mods, i also realoaded the mods (in terraria as well as terracustom)- yesterday i tried adound but there werent any settings for mod stuff like ores ect so i thought maybe i just cant choose for those but they doesent get generated as well (ive created some testing worlds with different settings but the new biomes and ores were missing).. today ive read there should be config files from terra custom for mods somewere but i couldnt find any (ive looked in any terraria and modloader realted files but coulndt find any) so i just tried to create another world and now it seems it cant even create worlds- it always gets stuck at the same spot at the creation loading screen *screenshot included*

now my question is, can i create worlds with mod ores and biomes? im not sure if my internet sourced informations are correct : 'D and im not very skilled at such things either- if yes, what could be the issue here? And what could possible the cause of it getting stuck so suddenly? (i didnt change anyting and i tried mutible generations but with always the same result)

![unbenannt](https://user-images.githubusercontent.com/34156970/33476625-ecc84ac2-d682-11e7-8dc0-50b72ce8fcdd.png)

Mod List:

Flying Minions Ignore  terrain
Unleveled
Wing Slot
Cheat Sheet
Leveled
Thorium
Customizable RespawnTime
Auto Trash
Wiki Search
Luminite Hammers
Wireless
Magic Storage

Game Version: 1.3.5.2
tModLoader: 0.10.1.1

(i hope ive got everything right xD and everything is understandable since english isnt my native language)

**Comment by JavidPack** [comment]
Yes, you can, but I'll look into this getting stuck on Corruption Pits thing

**Comment by I3IueJay** [comment]
ah nice thanks ^w^ :0 and what could be the cause that its not generating mod ores nor the mod biome within new generated worlds?

**Comment by merlinx2k** [comment]
got a similar problem, for me it gets stuck at Placing Traps.

these where the mods i used
  "JoostMod",
  "SacredTools",
  "ThoriumMod",
  "imkSushisMod",
  "WorldGenPreviewer"

i tried disabling them one by one but it did not help.
